.github/stale.yml

```diff
@@ -0,0 +1,18 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 15
+# Issues with these labels will never be considered stale
+exemptLabels:
```

Comment:
should we add the `2.0` label here as well?

Reply:
@danielsuguimoto I am thinking no... my sense is that we should move to have only short-lived issues here on GH, and move the broader roadmap/improvements to internal trackers. This is based on the stats that the `/issues` page is the second most view page after the homepage, and the overwhelming number of issues and different topics there does not help in the first impression for newcomers. This is a proposal though of course.

Reply:
@fabriciojs okidoki!